src/features/ui/select.tsx

```diff
@@ -1,29 +1,24 @@
-import { SelectOptions } from '../../types';
 import * as RadixSelect from '@radix-ui/react-select';
 import { useState } from 'react';
 import { FaChevronDown } from 'react-icons/fa';
+import { chainIdOptions } from '@/lib/chainIdOptions';
+import { useChainStore, useSelectStore } from '@/store/store';
 
-export const Select = ({
-  setIsEnableCheck,
-  setSelectedChainId,
-  selectOptions,
-}: {
-  setIsEnableCheck: (value: boolean) => void;
-  setSelectedChainId: (value: number) => void;
-  selectOptions: SelectOptions;
-}) => {
+export const Select = () => {
```

Comment:
Should be called ChainSelector as it can never be used for anything else given the states it contains.
I would prefer if this component did not have chain state in it in case you wanted to have another select in the app in future.